ChangeLog.md

```diff
@@ -20,6 +20,11 @@ See docs/process.md for more on how version tagging works.
 
 Current Trunk
 -------------
+- Fix generating of symbol files with `--emit-symbol-map` for ASM.js targets.
+  When `-s WASM=2` is used. Two symbols are generated:
+    - `[name].js.symbols` - storing WASM mapping
+    - `[name].wasm.js.symbols` - storing ASM.js mapping
+  In other cases a single `[name].js.symbols` file is created.
```

Comment:
The term "ASM.js" is perhaps confusing. `WASM=0`, `WASM=2` modes do not emit asm.js, although it is a little similar to it. I would replace all `ASM.js` with just `JS`.

Reply:
Ouh! I didn't know that, I've looked into files, and saw there is missing "use asm" - but I've assumed it's a part of modernization, that new browsers can detect such.